Implement major changes

dicodiaz opened the pull request:
- Install framer-motion package for animations & transitions
- Uninstall unnecessary packages (React Router & Redux)
- Add Google Analytics
- Add Scrollspy
- Add animations & transitions for all sections

DanteAlonsoHT commented on index.html:
```diff
@@ -186,6 +186,36 @@ <h1 class="box4 title4">About me</h1>
+                <li>
+                    <textarea name="msg" placeholder="Enter text here" maxlength="500" required></textarea>
+                </li>
+                <li>
+                    <button class="button" type="submit">Get in touch</button>
+                </li>
```
- [ ] [OPTIONAL] You can add the next CSS style for your button.
```
cursor: pointer;
```
In order to improve the details for UX 😉

DanteAlonsoHT reviewed (approved):
<h1>Status: Approve 🥇 ✔</h1>
Hi @dicodiaz  and @peterdgreat, 

### Your project is complete! I only give you one OPTIONAL change required, feel free to do it, (see the comments below 👇 ), and after that... it's time to merge it :shipit: 

## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

<img src="https://media.giphy.com/media/aNqEFrYVnsS52/giphy.gif" width="50%">

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dicodiaz committed:
Implement major changes:

- Install framer-motion package for animations & transitions
- Uninstall unnecessary packages (React Router & Redux)
- Add Google Analytics
- Add Scrollspy
- Add animations & transitions for all sections
